package.json

```diff
@@ -40,10 +40,10 @@
     "build": "tsc",
     "build:web": "webpack --config ./webpack/webpack.prod.js",
     "start": "node ./server",
-    "prettier": "prettier --write ./**/*.{js,ts,tsx,md}",
+    "prettier": "./node_modules/.bin/prettier --write ./**/*.{js,ts,tsx,md}",
     "test": "./node_modules/.bin/jest",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "eslint . --ext .ts,.tsx --fix",
+    "lint": "./node_modules/.bin/eslint . --ext .ts,.tsx",
+    "lint:fix": "./node_modules/.bin/eslint . --ext .ts,.tsx --fix",
```

Comment:
Do we really need this, since we already had pretter?

Reply:
we need it. ESLint to fix coding errors and Prettier to fix stylistic errors